src/Showtime.jsx

```diff
@@ -70,5 +70,5 @@ export default function Showtime({
         }
     }, [status, previousStatus, onHidden, onShowing]);
 
-    return isMounted ? children(ref) : null;
+    return isMounted ? cloneElement(Children.only(children), { ref }) : null;
```

Comment:
Also, if there's a better way to implement this than `cloneElement`, please let me know

Reply:
Full disclosure -- this is a shot in the dark not knowing much about the implementation limitations here...

Are [forward refs](https://reactjs.org/docs/forwarding-refs.html) or [HOCs](https://reactjs.org/docs/higher-order-components.html) the appropriate solution here?

Reply:
Forward ref isn't really relevant at this level. If the child you pass into `<Showtime/>` is a _component_, it must itself implement ref forwarding or else be wrapped in a div ([details here](https://github.com/azavea/react-showtime#attaching-the-ref)), so our ref can get attached properly. But that ref forwarding is entirely intrinsic to the child and can't be enforced or manipulated in any way from within the Showtime component.

As for HOC's, I don't _think_ they're relevant but truth be told I don't fully understand them because they always felt super hacky to me. If it came to a choice between HOC and render prop (if that choice is even relevant), I'd choose render prop in a heartbeat.

Reply:
HOCs [should not be used inside render](https://reactjs.org/docs/higher-order-components.html#dont-use-hocs-inside-the-render-method)

Reply:
👍

Reply:
This line should probably be adjusted to accommodate inbound refs, so they also get attached instead of overridden. [Here's how to do it](https://stackoverflow.com/questions/41876702/maintaining-component-refs-through-react-cloneelement/41877806#41877806). 

Related: [Instructions to developers for attaching their own refs](https://github.com/azavea/react-showtime#attaching-multiple-refs) along with the Showtime ref.